### ISSUE TITLE
Set armed mode without basestation not possible?

### ISSUE THREAD

**Issue by maglerod** [twrecked/hass-aarlo#806]
Using Arlo5 Pro Wifi without any Hub/base station.
Integration works fine except that i cannot use alarm panel with aarlo. 
So i cannot set armed mode to away or standby.

Can it be done somehow?

Regards
Andreas

**Comment by twrecked** [comment]
I have the same cameras and as long as you enable alarm control panel it should work.

How come you can't use the alarm panel? Are you getting errors when you try to enable it?

**Comment by maglerod** [comment]
Hi! I've tried to add the camera panel in configuration.yaml
Are there any other ways? 

This is how i've done (i've also tried to add it directly in configuration.yaml, no result).

configuration.yaml:
alarm_control_panel: !include alarm_control_panel_arlo.yaml

and then in my alarm_control_panel_arlo.yaml:
- platform: aarlo
  away_mode_name: armed
  home_mode_name: home
  night_mode_name: night
  disarmed_mode_name: disarmed
  trigger_time: 30
  alarm_volume: 8

Maybe it interfere with Alarmo?, 

I've noticed a lot of errors in log about the basestation: repeated every 5 minute
2023-09-30 22:07:12.637 ERROR (MainThread) [tibber.realtime] Watchdog: Connection is down, 2023-09-30 22:07:22.637183
2023-09-30 22:10:59.907 ERROR (MainThread) [pyaarlo] Could not find any base stations for device Framsida hus
2023-09-30 22:10:59.917 ERROR (MainThread) [pyaarlo] Could not find any base stations for device Pool
2023-09-30 22:10:59.921 ERROR (MainThread) [pyaarlo] Could not find any base stations for device Stallet
2023-09-30 22:10:59.925 ERROR (MainThread) [pyaarlo] Could not find any base stations for device Baksida hagar

**Comment by maglerod** [comment]
Hi, any clue whats wrong?. Really wish to be able to set armed mode. Thats more interested for me then watching the camera from HA.  I used to have Eufy cameras before, and i'm about going back to Eufy if i cant get this work :(

**Comment by maglerod** [comment]
> I have the same cameras and as long as you enable alarm control panel it should work.
> 
> How come you can't use the alarm panel? Are you getting errors when you try to enable it?

Hi, any clue whats wrong?. Really wish to be able to set armed mode. Thats more interested for me then watching the camera from HA. I used to have Eufy cameras before, and i'm about going back to Eufy if i cant get this work :(

**Comment by maglerod** [comment]
Are you really not using an Base station?, i keep having that error in my log constantly.

**Comment by maglerod** [comment]
Update! I installed an ArloQ Indoor Wifi camera , that camera appeared with alarm control panel automatically.
But still my other Arlo 5 Pro doesnt. Any idea what i can try?

**Comment by twrecked** [comment]
Can you post the model number of those cameras.

I'll take a look into the code and see why they aren't appearing.

**Comment by twrecked** [comment]
I have the Pro 4, not the Pro 5. If you send me the model number I can update the code.

**Comment by maglerod** [comment]
> I have the Pro 4, not the Pro 5. If you send me the model number I can update the code.

Hi, I'll try to attach the image and hope that can help.  Let me know else. I would be very very thankfull if it can be solved.


![arlopro5s](https://github.com/twrecked/hass-aarlo/assets/111062533/2c28f702-5061-48ca-9fb3-092778c2571c)

**Comment by twrecked** [comment]
Can you try beta 20?

**Comment by maglerod** [comment]
Yes, and it seemed to do the stuff! It works now, all cameras appeared and also the location.
I was able to arm / disarm via script! Exactly what i needed.

Thankyou so much for this!!

**Comment by Avatar1976** [comment]
BTW for what it is worth had the same problem but had worked around it with the following config (before b20).
![Screenshot_20231017_180327_Home Assistant](https://github.com/twrecked/hass-aarlo/assets/47725430/c8f23934-af0e-48a0-9519-653529ec4506)

This config below now does not work after b20.
About to put it back to the default as discussed above.
Just in case any other Pro5S only users have the same issue as me.

**Comment by Avatar1976** [comment]
Default config does not work for me as per (version b20) : 
![Screenshot_20231017_181428_Home Assistant](https://github.com/twrecked/hass-aarlo/assets/47725430/60045b02-dbf7-4f63-ae2d-304b5c1cb643)

Do I need to delete the session.pickle?

**Comment by Avatar1976** [comment]
Found a likely cause, is there something I have to do to make locations available to the account my home assistant accesses arlo with?
```
This error originated from a custom integration.

Logger: pyaarlo
Source: custom_components/aarlo/pyaarlo/__init__.py:722
Integration: Arlo Camera Support (documentation, issues)
First occurred: 18:40:44 (2 occurrences)
Last logged: 18:40:44

body-error=JSONDecodeError
No locations returned from /hmsdevicemanagement/users/3VCDPDK-114-60481659/locations
```

**Comment by Anders-Sandell** [comment]
Hi
I have the same issue as maglerod but with two essential 2 cameras. Is there something that has to be tweaked in the code for these as well?
Thanks!

**Comment by Anders-Sandell** [comment]
If @twrecked have a free slot to look at this my device model number: VMC3050A. Thanks!

Best,
Anders

**Comment by twrecked** [comment]
@Anders-Sandell Sorry, was busy. I'll add support for those cameras.

**Comment by Anders-Sandell** [comment]
@twrecked , no problem, let me know if you need more info from me.
